lib/elixir/lib/calendar/date.ex

```diff
@@ -398,6 +398,43 @@ defmodule Date do
     end
   end
 
+  @doc """
+  Converts a number of gregorian days to a `Date` struct.
+
+  ## Examples
+
+      iex> Date.from_gregorian_days(1)
+      ~D[0000-01-02]
+      iex> Date.from_gregorian_days(730_485)
+      ~D[2000-01-01]
+      iex> Date.from_gregorian_days(-1)
+      ~D[-0001-12-31]
+
+  """
+  @spec from_gregorian_days(integer(), Calendar.calendar()) :: t
+  def from_gregorian_days(days, calendar \\ Calendar.ISO) when is_integer(days) do
+    from_iso_days({days, 0}, calendar)
+  end
+
+  @doc """
+  Converts a `Date` struct to a number of gregorian days.
```

Comment:
since the function accepts not just Date structs but also NaiveDateTime etc, we should subtly change this here: (to. match e.g. Date.add/2 docs)

```suggestion
  Converts a `date` struct to a number of gregorian days.
```

Same goes for other similar places.